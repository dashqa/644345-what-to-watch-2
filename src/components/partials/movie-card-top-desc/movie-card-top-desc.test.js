import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardTopDesc from './movie-card-top-desc';

const props = {
  name: ``,
  genre: ``,
  released: 0,
};

it(`Movie card top desc correctly renders after relaunch`, () => {
  const tree = renderer.create(<MovieCardTopDesc {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
