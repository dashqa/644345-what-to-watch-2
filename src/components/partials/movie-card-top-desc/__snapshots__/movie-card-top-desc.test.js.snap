// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie card top desc correctly renders after relaunch 1`] = `
<div
  className="movie-card__desc"
>
  <h2
    className="movie-card__title"
  >
    
  </h2>
  <p
    className="movie-card__meta"
  >
    <span
      className="movie-card__genre"
    >
      
    </span>
    <span
      className="movie-card__year"
    >
      0
    </span>
  </p>
  <div
    className="movie-card__buttons"
  >
    <button
      className="btn movie-card__button btn-play"
      onClick={[Function]}
      type="button"
    >
      <svg
        height="19"
        viewBox="0 0 19 19"
        width="19"
      >
        <use
          xlinkHref="#play-s"
        />
      </svg>
      <span>
        Play
      </span>
    </button>
    <button
      className="btn movie-card__button btn-list"
      onClick={[Function]}
      type="button"
    >
      <svg
        height="20"
        viewBox="0 0 19 20"
        width="19"
      >
        <use
          xlinkHref="#add"
        />
      </svg>
      <span>
        My list
      </span>
    </button>
  </div>
</div>
`;
